client/src/main/java/com/paypal/selion/internal/reports/model/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Classes related web/mobile logging, extracting HTML source and taking screen shots to assist in test reporting.
```

Comment:
Please reword: suggest simplification: Something like: `Model classes to assist in test reporting. `